[2024-11-19T09:34:05.340+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-19T09:34:05.351+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [queued]>
[2024-11-19T09:34:05.357+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [queued]>
[2024-11-19T09:34:05.358+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-19T09:34:05.370+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): upsert_spotify_data> on 2024-11-18 12:00:00+00:00
[2024-11-19T09:34:05.377+0000] {standard_task_runner.py:72} INFO - Started process 184 to run task
[2024-11-19T09:34:05.380+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'bruno_mars', 'upsert_spotify_data', 'scheduled__2024-11-18T12:00:00+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmp9fnjeloo']
[2024-11-19T09:34:05.382+0000] {standard_task_runner.py:105} INFO - Job 62: Subtask upsert_spotify_data
[2024-11-19T09:34:05.396+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T09:34:05.422+0000] {task_command.py:467} INFO - Running <TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [running]> on host 368b98dccfcf
[2024-11-19T09:34:05.423+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T09:34:05.481+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bruno_mars' AIRFLOW_CTX_TASK_ID='upsert_spotify_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T12:00:00+00:00'
[2024-11-19T09:34:05.482+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-19T09:34:05.530+0000] {logging_mixin.py:190} INFO - Data artis berhasil diupdate.
[2024-11-19T09:34:05.535+0000] {logging_mixin.py:190} INFO - Data kota Jakarta berhasil diupdate.
[2024-11-19T09:34:05.536+0000] {logging_mixin.py:190} INFO - Data kota Bandung berhasil diupdate.
[2024-11-19T09:34:05.537+0000] {logging_mixin.py:190} INFO - Data kota Surabaya berhasil diupdate.
[2024-11-19T09:34:05.538+0000] {logging_mixin.py:190} INFO - Data kota Makassar berhasil diupdate.
[2024-11-19T09:34:05.538+0000] {logging_mixin.py:190} INFO - Data kota Semarang berhasil diupdate.
[2024-11-19T09:34:05.542+0000] {logging_mixin.py:190} INFO - Track Satu Bulan berhasil diupdate.
[2024-11-19T09:34:05.543+0000] {logging_mixin.py:190} INFO - Track Untungnya, Hidup Harus Tetap Berjalan berhasil diupdate.
[2024-11-19T09:34:05.544+0000] {logging_mixin.py:190} INFO - Track Kata Mereka Ini Berlebihan berhasil diupdate.
[2024-11-19T09:34:05.544+0000] {logging_mixin.py:190} INFO - Track Apa Mungkin berhasil diupdate.
[2024-11-19T09:34:05.545+0000] {logging_mixin.py:190} INFO - Track Kini Mereka Tahu berhasil diupdate.
[2024-11-19T09:34:05.546+0000] {logging_mixin.py:190} INFO - Track Lama-Lama berhasil diupdate.
[2024-11-19T09:34:05.546+0000] {logging_mixin.py:190} INFO - Track Masa Sepi berhasil diupdate.
[2024-11-19T09:34:05.547+0000] {logging_mixin.py:190} INFO - Track Sialnya, Hidup Harus Tetap Berjalan berhasil diupdate.
[2024-11-19T09:34:05.549+0000] {logging_mixin.py:190} INFO - Track Terlintas berhasil diupdate.
[2024-11-19T09:34:05.550+0000] {logging_mixin.py:190} INFO - Track Sinyal-Sinyal berhasil diupdate.
[2024-11-19T09:34:05.554+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-19T09:34:05.562+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-19T09:34:05.563+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=bruno_mars, task_id=upsert_spotify_data, run_id=scheduled__2024-11-18T12:00:00+00:00, execution_date=20241118T120000, start_date=20241119T093405, end_date=20241119T093405
[2024-11-19T09:34:05.594+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-19T09:34:05.606+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-19T09:34:05.608+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-11-19T09:51:07.636+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-19T09:51:07.647+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [queued]>
[2024-11-19T09:51:07.654+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [queued]>
[2024-11-19T09:51:07.655+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-19T09:51:07.669+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): upsert_spotify_data> on 2024-11-18 12:00:00+00:00
[2024-11-19T09:51:07.674+0000] {standard_task_runner.py:72} INFO - Started process 260 to run task
[2024-11-19T09:51:07.676+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'bruno_mars', 'upsert_spotify_data', 'scheduled__2024-11-18T12:00:00+00:00', '--job-id', '90', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpftrs45lr']
[2024-11-19T09:51:07.679+0000] {standard_task_runner.py:105} INFO - Job 90: Subtask upsert_spotify_data
[2024-11-19T09:51:07.694+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T09:51:07.722+0000] {task_command.py:467} INFO - Running <TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [running]> on host 84d1fd291955
[2024-11-19T09:51:07.724+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T09:51:07.788+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bruno_mars' AIRFLOW_CTX_TASK_ID='upsert_spotify_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T12:00:00+00:00'
[2024-11-19T09:51:07.789+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-19T09:51:07.860+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/dag2.py", line 266, in upsert_spotify_data
    with open('/opt/airflow/bruno_mars_spotify.json', 'r') as json_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/bruno_mars_spotify.json'
[2024-11-19T09:51:07.869+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=bruno_mars, task_id=upsert_spotify_data, run_id=scheduled__2024-11-18T12:00:00+00:00, execution_date=20241118T120000, start_date=20241119T095107, end_date=20241119T095107
[2024-11-19T09:51:07.880+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-19T09:51:07.880+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 90 for task upsert_spotify_data ([Errno 2] No such file or directory: '/opt/airflow/bruno_mars_spotify.json'; 260)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/dag2.py", line 266, in upsert_spotify_data
    with open('/opt/airflow/bruno_mars_spotify.json', 'r') as json_file:
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/opt/airflow/bruno_mars_spotify.json'
[2024-11-19T09:51:07.891+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-11-19T09:51:07.902+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-19T09:51:07.903+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-11-19T09:56:32.209+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-19T09:56:32.223+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [queued]>
[2024-11-19T09:56:32.233+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [queued]>
[2024-11-19T09:56:32.234+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-19T09:56:32.249+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): upsert_spotify_data> on 2024-11-18 12:00:00+00:00
[2024-11-19T09:56:32.256+0000] {standard_task_runner.py:72} INFO - Started process 162 to run task
[2024-11-19T09:56:32.259+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'bruno_mars', 'upsert_spotify_data', 'scheduled__2024-11-18T12:00:00+00:00', '--job-id', '99', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpv_q4okji']
[2024-11-19T09:56:32.262+0000] {standard_task_runner.py:105} INFO - Job 99: Subtask upsert_spotify_data
[2024-11-19T09:56:32.281+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T09:56:32.311+0000] {task_command.py:467} INFO - Running <TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [running]> on host 6aba41e3ae6f
[2024-11-19T09:56:32.313+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T09:56:32.372+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bruno_mars' AIRFLOW_CTX_TASK_ID='upsert_spotify_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T12:00:00+00:00'
[2024-11-19T09:56:32.373+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-19T09:56:32.426+0000] {logging_mixin.py:190} INFO - Data artis berhasil dimasukkan.
[2024-11-19T09:56:32.430+0000] {logging_mixin.py:190} INFO - Data kota Jakarta berhasil dimasukkan.
[2024-11-19T09:56:32.431+0000] {logging_mixin.py:190} INFO - Data kota São Paulo berhasil dimasukkan.
[2024-11-19T09:56:32.433+0000] {logging_mixin.py:190} INFO - Data kota Quezon City berhasil dimasukkan.
[2024-11-19T09:56:32.434+0000] {logging_mixin.py:190} INFO - Data kota Santiago berhasil dimasukkan.
[2024-11-19T09:56:32.435+0000] {logging_mixin.py:190} INFO - Data kota Bandung berhasil dimasukkan.
[2024-11-19T09:56:32.439+0000] {logging_mixin.py:190} INFO - Track Die With A Smile berhasil dimasukkan.
[2024-11-19T09:56:32.440+0000] {logging_mixin.py:190} INFO - Track APT. berhasil dimasukkan.
[2024-11-19T09:56:32.442+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/mysql/connector/connection_cext.py", line 706, in cmd_query
    self._cmysql.query(
_mysql_connector.MySQLInterfaceError: Out of range value for column 'streamCount' at row 1

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/dag2.py", line 271, in upsert_spotify_data
    upsert_top_tracks(connection, data)
  File "/opt/airflow/dags/dag2.py", line 157, in upsert_top_tracks
    cursor.execute(query, track_data)
  File "/home/airflow/.local/lib/python3.12/site-packages/mysql/connector/cursor_cext.py", line 357, in execute
    result = self._connection.cmd_query(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py", line 106, in wrapper
    result = method(cnx, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/mysql/connector/connection_cext.py", line 714, in cmd_query
    raise get_mysql_exception(
mysql.connector.errors.DataError: 1264 (22003): Out of range value for column 'streamCount' at row 1
[2024-11-19T09:56:32.454+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=bruno_mars, task_id=upsert_spotify_data, run_id=scheduled__2024-11-18T12:00:00+00:00, execution_date=20241118T120000, start_date=20241119T095632, end_date=20241119T095632
[2024-11-19T09:56:32.467+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-19T09:56:32.467+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 99 for task upsert_spotify_data (1264 (22003): Out of range value for column 'streamCount' at row 1; 162)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/mysql/connector/connection_cext.py", line 706, in cmd_query
    self._cmysql.query(
_mysql_connector.MySQLInterfaceError: Out of range value for column 'streamCount' at row 1

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 256, in execute_callable
    return runner.run(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/dag2.py", line 271, in upsert_spotify_data
    upsert_top_tracks(connection, data)
  File "/opt/airflow/dags/dag2.py", line 157, in upsert_top_tracks
    cursor.execute(query, track_data)
  File "/home/airflow/.local/lib/python3.12/site-packages/mysql/connector/cursor_cext.py", line 357, in execute
    result = self._connection.cmd_query(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/mysql/connector/opentelemetry/context_propagation.py", line 106, in wrapper
    result = method(cnx, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/mysql/connector/connection_cext.py", line 714, in cmd_query
    raise get_mysql_exception(
mysql.connector.errors.DataError: 1264 (22003): Out of range value for column 'streamCount' at row 1
[2024-11-19T09:56:32.513+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-11-19T09:56:32.525+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-19T09:56:32.526+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-11-19T10:00:39.970+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-19T10:00:39.981+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [queued]>
[2024-11-19T10:00:39.987+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [queued]>
[2024-11-19T10:00:39.988+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-19T10:00:40.000+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): upsert_spotify_data> on 2024-11-18 12:00:00+00:00
[2024-11-19T10:00:40.005+0000] {standard_task_runner.py:72} INFO - Started process 272 to run task
[2024-11-19T10:00:40.008+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'bruno_mars', 'upsert_spotify_data', 'scheduled__2024-11-18T12:00:00+00:00', '--job-id', '107', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpq11o1o2o']
[2024-11-19T10:00:40.010+0000] {standard_task_runner.py:105} INFO - Job 107: Subtask upsert_spotify_data
[2024-11-19T10:00:40.025+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T10:00:40.050+0000] {task_command.py:467} INFO - Running <TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [running]> on host 6aba41e3ae6f
[2024-11-19T10:00:40.052+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T10:00:40.120+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bruno_mars' AIRFLOW_CTX_TASK_ID='upsert_spotify_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T12:00:00+00:00'
[2024-11-19T10:00:40.120+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-19T10:00:40.170+0000] {logging_mixin.py:190} INFO - Data artis berhasil diupdate.
[2024-11-19T10:00:40.174+0000] {logging_mixin.py:190} INFO - Data kota Jakarta berhasil diupdate.
[2024-11-19T10:00:40.175+0000] {logging_mixin.py:190} INFO - Data kota São Paulo berhasil diupdate.
[2024-11-19T10:00:40.176+0000] {logging_mixin.py:190} INFO - Data kota Quezon City berhasil diupdate.
[2024-11-19T10:00:40.177+0000] {logging_mixin.py:190} INFO - Data kota Santiago berhasil diupdate.
[2024-11-19T10:00:40.178+0000] {logging_mixin.py:190} INFO - Data kota Bandung berhasil diupdate.
[2024-11-19T10:00:40.181+0000] {logging_mixin.py:190} INFO - Track Die With A Smile berhasil dimasukkan.
[2024-11-19T10:00:40.182+0000] {logging_mixin.py:190} INFO - Track APT. berhasil dimasukkan.
[2024-11-19T10:00:40.183+0000] {logging_mixin.py:190} INFO - Track Locked out of Heaven berhasil dimasukkan.
[2024-11-19T10:00:40.184+0000] {logging_mixin.py:190} INFO - Track Just the Way You Are berhasil dimasukkan.
[2024-11-19T10:00:40.185+0000] {logging_mixin.py:190} INFO - Track That's What I Like berhasil dimasukkan.
[2024-11-19T10:00:40.185+0000] {logging_mixin.py:190} INFO - Track When I Was Your Man berhasil dimasukkan.
[2024-11-19T10:00:40.186+0000] {logging_mixin.py:190} INFO - Track It Will Rain berhasil dimasukkan.
[2024-11-19T10:00:40.188+0000] {logging_mixin.py:190} INFO - Track Grenade berhasil dimasukkan.
[2024-11-19T10:00:40.188+0000] {logging_mixin.py:190} INFO - Track Talking to the Moon berhasil dimasukkan.
[2024-11-19T10:00:40.189+0000] {logging_mixin.py:190} INFO - Track Treasure berhasil dimasukkan.
[2024-11-19T10:00:40.192+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-19T10:00:40.199+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-19T10:00:40.200+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=bruno_mars, task_id=upsert_spotify_data, run_id=scheduled__2024-11-18T12:00:00+00:00, execution_date=20241118T120000, start_date=20241119T100039, end_date=20241119T100040
[2024-11-19T10:00:40.221+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-19T10:00:40.232+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-19T10:00:40.233+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-11-19T10:03:12.699+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-19T10:03:12.710+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [queued]>
[2024-11-19T10:03:12.717+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [queued]>
[2024-11-19T10:03:12.717+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-19T10:03:12.729+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): upsert_spotify_data> on 2024-11-18 12:00:00+00:00
[2024-11-19T10:03:12.733+0000] {standard_task_runner.py:72} INFO - Started process 376 to run task
[2024-11-19T10:03:12.738+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'bruno_mars', 'upsert_spotify_data', 'scheduled__2024-11-18T12:00:00+00:00', '--job-id', '115', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpst2qb0bm']
[2024-11-19T10:03:12.740+0000] {standard_task_runner.py:105} INFO - Job 115: Subtask upsert_spotify_data
[2024-11-19T10:03:12.760+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T10:03:12.786+0000] {task_command.py:467} INFO - Running <TaskInstance: bruno_mars.upsert_spotify_data scheduled__2024-11-18T12:00:00+00:00 [running]> on host 6aba41e3ae6f
[2024-11-19T10:03:12.788+0000] {logging_mixin.py:190} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-11-19T10:03:12.846+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='bruno_mars' AIRFLOW_CTX_TASK_ID='upsert_spotify_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-18T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-18T12:00:00+00:00'
[2024-11-19T10:03:12.846+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-19T10:03:12.905+0000] {logging_mixin.py:190} INFO - Data artis berhasil diupdate.
[2024-11-19T10:03:12.911+0000] {logging_mixin.py:190} INFO - Data kota Jakarta berhasil diupdate.
[2024-11-19T10:03:12.912+0000] {logging_mixin.py:190} INFO - Data kota São Paulo berhasil diupdate.
[2024-11-19T10:03:12.913+0000] {logging_mixin.py:190} INFO - Data kota Quezon City berhasil diupdate.
[2024-11-19T10:03:12.914+0000] {logging_mixin.py:190} INFO - Data kota Santiago berhasil diupdate.
[2024-11-19T10:03:12.915+0000] {logging_mixin.py:190} INFO - Data kota Bandung berhasil diupdate.
[2024-11-19T10:03:12.919+0000] {logging_mixin.py:190} INFO - Track Die With A Smile berhasil diupdate.
[2024-11-19T10:03:12.920+0000] {logging_mixin.py:190} INFO - Track APT. berhasil diupdate.
[2024-11-19T10:03:12.921+0000] {logging_mixin.py:190} INFO - Track Locked out of Heaven berhasil diupdate.
[2024-11-19T10:03:12.922+0000] {logging_mixin.py:190} INFO - Track Just the Way You Are berhasil diupdate.
[2024-11-19T10:03:12.923+0000] {logging_mixin.py:190} INFO - Track That's What I Like berhasil diupdate.
[2024-11-19T10:03:12.924+0000] {logging_mixin.py:190} INFO - Track When I Was Your Man berhasil diupdate.
[2024-11-19T10:03:12.925+0000] {logging_mixin.py:190} INFO - Track It Will Rain berhasil diupdate.
[2024-11-19T10:03:12.926+0000] {logging_mixin.py:190} INFO - Track Grenade berhasil diupdate.
[2024-11-19T10:03:12.927+0000] {logging_mixin.py:190} INFO - Track Talking to the Moon berhasil diupdate.
[2024-11-19T10:03:12.928+0000] {logging_mixin.py:190} INFO - Track Treasure berhasil diupdate.
[2024-11-19T10:03:12.931+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-19T10:03:12.938+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-19T10:03:12.939+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=bruno_mars, task_id=upsert_spotify_data, run_id=scheduled__2024-11-18T12:00:00+00:00, execution_date=20241118T120000, start_date=20241119T100312, end_date=20241119T100312
[2024-11-19T10:03:12.990+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-19T10:03:13.002+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-19T10:03:13.003+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
